DEVELOP.md

```diff
@@ -0,0 +1,252 @@
+# Chaos Mesh Development Guide
+Chaos Mesh currently have many types of chaos such as IOChaos, NetworkChaos, PodChaos. This guide aims to introduce the way to add new chaos type for Chaos Mesh, which includes setting up the development environment and adding a hello-world chaos type.
+
+## Setting Up The Development Environment
+### Requirements
+You should have these components installed in your system:
+- golang (>= v1.13), if your golang version < v1.13, you can use golang version manager such as [gvm](https://github.com/moovweb/gvm) to switch to a newer one.
+- [yarn](https://yarnpkg.com/lang/en/) and [nodejs](https://nodejs.org/en/): for chaos-dashboard
```

Comment:
maybe we can add `[Option]` here for dashboard.

Reply:
The yarn and nodejs is for the dashboard, they are optional, but recommend because if we don't compile dashboard, we must compile each other component one by one(many `make xxx` instead of a single `make`), which need more knowledge about the architecture of chaos-mesh